app/controllers/apartments_controller.rb

```diff
@@ -1,2 +1,6 @@
 class ApartmentsController < ApplicationController
+    def index
```

Comment:
You should not need to have an index method to write tests and validations.  The index method is on a separate card.